source/extensions/omni.isaac.lab/omni/isaac/lab/terrains/terrain_generator_cfg.py

```diff
@@ -105,8 +105,7 @@ class TerrainGeneratorCfg:
     """Configuration for the terrain generator."""
 
     seed: int | None = None
-    """The seed for the random number generator. Defaults to None,
-    in which case the seed is not set."""
+    """The seed for the random number generator. Defaults to None, in which case the seed is not set."""
```

Comment:
Would the terrain seed override the env seed if they are both set?

Reply:
Env seed overwrites this one if the configuration handles it.